client/package.json

```diff
@@ -22,7 +22,7 @@
   "description": "A better commenting experience from Vox Media.",
   "scripts": {
     "build:client": "NODE_OPTIONS=\"--openssl-legacy-provider --no-experimental-fetch\" ts-node --transpile-only ./scripts/build.ts",
-    "build:development": "NODE_OPTIONS=\"--openssl-legacy-provider --no-experimental-fetch\" NODE_ENV=development pnpm run --parallel generate build:client",
+    "build:development": "NODE_OPTIONS=\"--openssl-legacy-provider --no-experimental-fetch\" NODE_ENV=development pnpm run generate && pnpm run build:client",
```

Comment:
this fixes the dev client builds not actually compiling (transpiling) the client TS code and it only doing the generate steps